### PR TITLE
UnixPB: Add Ubuntu 2104 to DockerStatic role

### PR DESCRIPTION
**Pull request by Haroon-Khel**
ref #2143

### CODE REVIEW

**Review by sxa — COMMENTED** [review]
The [original PR](https://github.com/adoptium/infrastructure/pull/2188) said `Will test on a machine shortly` - have you done that now?

**Comment by Haroon-Khel** [comment]
Changes run fine. Ubuntu 2104 docker image builds fine too. 

Based on what was already in the DockerStatic role, I used the port map 2226:22 for the ubuntu 2104 image. Testing on dockerhost-equinix-ubuntu2004-x64-1, (https://ci.adoptopenjdk.net/computer/test-docker-ubuntu2004-x64-1/), that port was already in use by a centos 8 docker container so I used a different port for testing. 

My question is should I keep it as 2226 in the playbooks?

**Comment by sxa** [comment]
> My question is should I keep it as 2226 in the playbooks?

Honestly, since it's not an LTS release I'd either select a new one later down the list, or replace the port defined for 20.10 with this one and remove 20.10 from the role.

**Comment by Haroon-Khel** [comment]
>Honestly, since it's not an LTS release I'd either select a new one later down the list, or replace the port defined for 20.10 with this one and remove 20.10 from the role.

Im in favour of the latter

File: ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/DockerStatic/tasks/main.yml [hunk 1]
```diff
@@ -62,8 +62,8 @@
   tags: startcontainers
   ignore_errors: yes
 
-- name: Start Ubuntu 20.10 container if not already started
-  command: docker run --restart unless-stopped -p 2225:22 --cpus=4.0 --memory=6G --detach --name U2010_2225 aqa_u2010
+- name: Start Ubuntu 21.04 container if not already started
+  command: docker run --restart unless-stopped -p 2225:22 --cpus=4.0 --memory=6G --detach --name U2104_2226 aqa_u2104
```
Comment (sxa) [inline ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/DockerStatic/tasks/main.yml:66]:
```suggestion
  command: docker run --restart unless-stopped -p 2225:22 --cpus=4.0 --memory=6G --detach --name U2104_2225 aqa_u2104
```

**Review by sxa — CHANGES_REQUESTED** [review]
Let's ensure the machine name is consistent with the port number

File: ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/DockerStatic/tasks/main.yml (hunk 1 above)
Comment (Haroon-Khel) [reply ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/DockerStatic/tasks/main.yml:66]: Ah I knew there was something I missed

**Review by Willsparker — APPROVED** [review]
LGTM :+1:

**Review by sxa — APPROVED** [review]
LGTM - Thanks!

**Pull request merged**